src/reducers/downloadFileReducer.js

```diff
@@ -0,0 +1,21 @@
+import { FILE_DOWNLOAD_LINK } from '../actions/downloadFileActions';
+
+const initialState = {
+    downloadFile: false
+};
+
+export default function downloadFileReducer( state = initialState, action ) {
+    switch(action.type) {
```

Comment:
It is not mandatory to use switch all-time in reducer, in here there is only two cases you can simply use if-else instead of switch